content/working_with/console/widgets/userManagement.md

```diff
@@ -23,9 +36,56 @@ The hamburger menu on the right of every tenant allows performing the following
 * **Deleting the user** - possible only if the user does not belong to any groups, assigned to any tenants and is the creator of any resources on the manager. 
  
 Also, using the “Add” button on the right top corner of the widget, you will be able to create new users.
-Please notice that if you choose to  authenticate the users in front of an external user management system, you will not be able to create or delete the users in cloudify, nor to assigned them to Cloudify user groups,  to prevent conflicts between the two systems which might cause security problems. 
+Please notice that if you choose to authenticate the users in front of an external user management system, you will not be able to create or delete the users in cloudify, nor to assigned them to Cloudify user groups,  to prevent conflicts between the two systems which might cause security problems. 
```

Comment:
```suggestion
Please notice that if you choose to authenticate the users in front of an external user management system, you will not be able to create or delete the users in Cloudify, nor to assigned them to Cloudify user groups,  to prevent conflicts between the two systems which might cause security problems. 
```

Reply:
Please notice that if you choose to use external authentication for users (e.g. LDAP), creation or deletion of local users, as well as assignments of local users with groups are not allowed to prevent conflicts between the two systems which might cause security problems.